docs/src/main/sphinx/installation/cli.rst

```diff
@@ -29,6 +29,22 @@ The CLI uses the HTTP protocol and the
 :doc:`Trino client REST API </develop/client-protocol>` to communicate
 with Trino.
 
+TLS/HTTPS
+---------
+
+Typically a Trino server is available with a HTTPS URL, so all network traffic
+from the CLI uses TLS. This :doc:`TLS configuration </security/tls>` is common,
```

Comment:
"A Trino server is typically available..." vs starting with "typically"

Reply:
Updated